vicodyn/src/gateway/vicodyn.cpp

```diff
@@ -133,25 +116,47 @@ vicodyn_t::vicodyn_t(context_t& _context, const std::string& _local_uuid, const
         dynamic_t result = dynamic_t::empty_object;
         dynamic_t& app_result = result.as_object()["apps"];
         peers.apply_shared([&](const vicodyn::peers_t::data_t& data) mutable {
+            auto convert_app_services = [&](const vicodyn::peers_t::app_services_t& from) {
+                dynamic_t app_services_result = dynamic_t::empty_object;
+                for (const auto& app_service : from) {
+                    dynamic_t& peer_result = app_services_result.as_object()[app_service.first];
+
+                    auto banned_still_ns = app_service.second.banned_still().count();
+                    if (banned_still_ns > 0) {
+                        peer_result.as_object()["banned_ms"] = banned_still_ns / 1000000;
```

Comment:
Is it (here and later in code) pure integer division? Would be millisecond fractions also be of some interest?

Reply:
I think for `banned` it is not matter.